dev-tools.asciidoc

```diff
@@ -367,13 +367,30 @@ Website link; https://openzeppelin.org/
 
 Read the docs link; http://zeppelin-solidity.readthedocs.io/en/latest/index.html
 
-=== ZeppelinOS
-zeppelin_os is an open-source, distributed platform of tools and services on top of the EVM
-to develop and manage smart contract applications securely.
+=== zeppelin_os
+https://github.com/zeppelinos[zeppelin_os] is an open-source, distributed platform of tools and services on top of the EVM
+to develop and manage smart contract applications securely. 
 
-Github link; https://github.com/zeppelinos
+Unlike OpenZeppelin's code, which needs to be re-deployed with each application every time, zeppelin_os's code lives on-chain. Applications that need a given functionality, say, an ERC20 token, not only don't have to re-design and re-audit its implementation (something that OpenZeppelin solved) but don't even need to deploy it. With zeppelin_os, an application interacts with the token's on-chain implementation directly, much in the same way a desktop application interacts with the components of its underlaying OS.
+
+Applications that make use of OpenZeppelin avoid having to "reinvent the wheel" by reusing the library's curated and peer reviewed contracts. However, every single time an application uses an ERC20 token implementation, the same ERC20 bytecode is deployed into the blockchain, over and over again. Such bytecode per-se, exists duplicated in the network countless times. Now, applications that make use of zeppelin_os avoid this unnecessary duplication. Instead of deploying their own ERC20 implementation, they link to a contract that defines the latest ERC20 implementation accepted by the community. This single, central implementation is deployed in the blockchain only once, much like Solidity's libraries, but are considerably more sophisticated.
```

Comment:
remove: "per-se,":
Such bycode exists duplicated..

Reply:
Dammit. I thought I was being smart by using a bit of latin.